### DISCUSSION TITLE
这是发生了什么问题？

### DISCUSSION THREAD

**Discussion by mugehan** [sayaka-sh/spmp#409] [Q&A]
![Screenshot_20241205_151700.jpg](https://github.com/user-attachments/assets/6a0a7b75-4050-4326-aa51-9b32032b2958)

![Screenshot_20241205_175803.jpg](https://github.com/user-attachments/assets/62c05851-dd51-4eb4-8720-62209294e5e9)

**Comment by sayaka-sh** [comment]
Please create bug reports for each separate issue.

**Reply by mugehan** [reply]
I don't know what happened, I don't understand, but it just happened

**Reply by mugehan** [reply]
第一个是从云端下载歌曲列表弹出来的错误，后面那个全屏是突然打开app就弹出来的，看不懂bug代码。之前就有过这个问题，但是找不到哪里反馈，今天看到这里可以对话，我就发这里了，一直等着更新，看到没动静都有点放弃了。
